### ISSUE TITLE
Few questions on implements

### ISSUE THREAD

**Issue by hun-hub** [Iceage7/SL2A-INR#1]
Hi, thanks for the impressive work, and congrats on the ICCV 2025 acceptance. I have a few questions about your NeRF implementation. What network depth and width did you run with in terms of the number of hidden layers and hidden features? What polynomial degree K did you use for the Learnable Activation block? What rank did you set for the low-rank linear layers? What learning rate did you use during training? Was your implementation based on torch-ngp (https://github.com/ashawkey/torch-ngp), and if so were there any notable modifications? Thanks.

**Comment by Iceage7** [comment]
Hi, thanks a lot for your interest and kind words!

For training NeRF, we adapted the training code from [https://github.com/liuzhen0212/FINER/tree/main/nerf](https://github.com/liuzhen0212/FINER/tree/main/nerf)
, which is based on torch-ngp, by modifying the NeRF network architecture. We used rank = 32 for the sigma_net and rank = 64 for the color_net.

We found that using a Chebyshev layer as the initial layer of the sigma_net with degree = 64, while keeping all other layers as low-rank or standard linear layers, provided good performance. We also noticed slightly better results when skip connections were applied only to the sigma_net. The learning rate was set to 1e-3.

The full network architecture, including depth and width, is shown below. The total number of trainable parameters was 174,360.

NeRFNetwork(
  (sigma_net): ModuleList(
    (0): ChebyLayer(
      (cheby): ChebyKANLayer()
      (norm): LayerNorm((182,), eps=1e-05, elementwise_affine=True)
    )
    (1-2): 2 x LowRankReLULayer()
    (3): LinearLayer(
      (linear): Linear(in_features=182, out_features=183, bias=True)
    )
  )
  (color_net): ModuleList(
    (0): LinearLayer(
      (linear): Linear(in_features=185, out_features=182, bias=True)
    )
    (1-2): 2 x LowRankReLULayer()
    (3): LinearLayer(
      (linear): Linear(in_features=182, out_features=3, bias=True)
    )
  )
)
